dotCMS/src/main/java/com/dotcms/uuid/shorty/ShortyIdAPIImpl.java

```diff
@@ -143,17 +147,17 @@ public String shortInodeUri(Contentlet c) {
     return null;
   }
 
-    public void validShorty(final String test) {
-        if (test == null || test.length() < MINIMUM_SHORTY_ID_LENGTH || test.length()>36) {
-            throw new ShortyException(
-                    "shorty " + test + " is not a short id.  Short Ids should be " + MINIMUM_SHORTY_ID_LENGTH + " chars in length");
-        }
-        
-        for (char c : test.toCharArray()) {
-            if (!((c >= 'a' && c <= 'z') || (c >= 'A' && c <= 'Z') || (c >= '0' && c <= '9') || c=='-')) {
-                throw new ShortyException(
-                        "shorty " + test + " is not an alpha numeric id.  Short Ids should be " + MINIMUM_SHORTY_ID_LENGTH + " alpha/numeric chars in length");
-            }
-        }
-    }
-}
+  public void validShorty(final String test) {
+      if ((test == null || test.length() < MINIMUM_SHORTY_ID_LENGTH || test.length()>36) && !ENABLE_SUPPORT_LEGACY_IDS) {
+          throw new ShortyException(
+                  "Short Id is invalid. Valid Length of Short Ids should be " + MINIMUM_SHORTY_ID_LENGTH + " chars. Short Id is: " + test!=null?test:"null");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 157 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)